_docs-2/getting-started/features.md

```diff
@@ -16,23 +16,23 @@ redirect_from:
 
 ### Iterators
 
-A server-side programming mechanism to encode functions such as filtering and
+[Iterators] are a server-side programming mechanism that encode functions such as filtering and
 aggregation within the data management steps (scopes where data is read from or
 written to disk) that happen in the tablet server.
 
-### Cell labels
+### Security labels
 
-An additional portion of the Key that sorts after the column qualifier and
-before the timestamp. It is called column visibility and enables expressive
-cell-level access control. Authorizations are passed with each query to control
-what data is returned to the user. The column visibilities are boolean AND and
-OR combinations of arbitrary strings (such as "(A&amp;B)|C") and authorizations
-are sets of strings (such as {C,D}).
+Accumulo Keys can contain a [security label]({% durl security/labels %})
+(called a Column Visibility) that enables expressive cell-level access control.
+Authorizations are passed with each query to control what data is returned to the user.
+Column visibilities support boolean `AND` and `OR` combinations of arbitrary strings (such
+as `(A&B)|C`) and authorizations are sets of strings (such as `{C,D}`).
 
 ### Constraints
 
-Configurable conditions under which writes to a table will be rejected.
-Constraints are written in Java and configurable on a per table basis.
+[Constraints]({% durl getting-started/table_configuration#constraints %}) are configurable
+conditions under which writes to a table will be rejected. Constraints are written in Java
```

Comment:
Could make this more clear by saying something like... "Constraints are configurable conditions that allow table writes to be rejected."